terraform/hcl/applicable/req_sw_terraform_aws_batch_no_privileged.tf

```diff
@@ -0,0 +1,39 @@
+resource "aws_batch_job_definition" "vulnerable_example" {
+  name = "tf_test_batch_job_definition"
+  type = "container"
+
+  container_properties = <<CONTAINER_PROPERTIES
```

Comment:
[comment]: <> (FrogbotReviewComment)


## 🛠️ Infrastructure as Code Vulnerability
<div align='center'>

| Severity                | Finding                  |
| :---------------------: | :-----------------------------------: |
| ![](https://raw.githubusercontent.com/jfrog/frogbot/master/resources/v2/applicableHighSeverity.png)<br>    High | privileged=True was detected |

</div>

<details>
<summary> <b>Full description</b> </summary>
<br>

Resource `aws_batch_job_definition` should not set `container_properties.privileged=true` (default value is false)

Vulnerable example -
```
resource "aws_batch_job_definition" "vulnerable_example" {
  container_properties = <<CONTAINER_PROPERTIES
{
    "privileged": true,
}
CONTAINER_PROPERTIES
}
```

Secure example -
```
resource "aws_batch_job_definition" "secure_example" {
  container_properties = <<CONTAINER_PROPERTIES
{
    "privileged": false,  # or not set
}
CONTAINER_PROPERTIES
}
```

</details>


---
<div align='center'>

[🐸 JFrog Frogbot](https://docs.jfrog-applications.jfrog.io/jfrog-applications/frogbot)

</div>